doc/neps/nep-0036-fair-play.rst

```diff
@@ -0,0 +1,150 @@
+==================
+NEP 36 — Fair Play
+==================
+
+:Author: Stéfan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Standards Track
+:Created: 2019-10-24
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This document describes Rules of Play for engaging with the NumPy
+project as an external entity.  It discusses usage of the name NumPy,
+modified distributions of NumPy, and how to best communicate
+with the project when patched versions are deployed.
+
+Companies and developers will know after reading this NEP what kinds
+of behavior the NumPy community consider troublesome, bothersome, and
+unacceptable.
+
+Motivation
+----------
+
+In December 2018, through the filing of `Issue #12515
+<https://github.com/numpy/numpy/issues/12515>`__, it became known to
+the NumPy project that an external company had published a version of
+NumPy with an extended footprint.  In this specific instance, a bug
+was filed against the `np.erf` function---a function that NumPy itself
+had never provided.
+
+In the months that followed, several conversations was had with the
+entity involved, and a better understanding emerged of needs on either
+side.  The entity agreed to publish the patches made against Python,
+and a special label was added on the NumPy issue tracker to identify
+issues pertaining to this specific distribution.
+
+However, this is *not* the only case where modified versions of NumPy
+are being made available, and it causes wasted time for our
+developers: identifying the release from which the error came, how the
+release was modified from the version we provide, and whether we are
+able to address the issue (if not, where to relay the report to).
+
+During a community call on XX/XX/XX the community decided to draft a
+set of guidelines for companies and external developers to use when
+they modify and redistribute NumPy.
+
+Scope
+-----
+
+This document aims to define a minimal set of rules that, when
+followed, will placate the NumPy developers.  Our hope, however, is
+that developers who think that they need to modify NumPy will consider
+contributing to the project, or using one of the many mechanisms we
+have for allowing our APIs to operate on externally defined objects.
+
+We encourage and exhort developers to first discuss their problem with
+the NumPy developers on the `mailing list <>`__ before resorting to
+patching and releasing their own version of the library.
+
+Fair Play Rules
+---------------
+
+1. Do not reuse the NumPy name for projects not developed by the NumPy
+   community.
+
+   At time of writing, there are only two `numpy`-named packages
+   developed by the community: `numpy` and `numpy-financial`.  We ask
```

Comment:
Not entirely true, e.g. `rnumpy` and `unumpy`. Those are meant for inclusion into the main numpy package, moved under the NumPy GitHub org, or similar. There are many other packages with `numpy` in the name: https://pypi.org/search/?q=numpy. Most of those indeed look undesirable.